.copier-answers.yml

```diff
@@ -17,6 +17,7 @@ org_name: Odoo Community Association (OCA)
 org_slug: OCA
 rebel_module_groups:
 - stock_picking_to_batch_group_fields
+- stock_customer_deposit
```

Comment:
@EmilioPascual Is this really necessary ? Doesn't your module require a configuration parameter in order to activate a non standard behaviour ?